### PR TITLE
Update Chinese translation for 1.21.4

### PR DESCRIPTION
**Pull request by Genglxy**
Update to the latest version.
Fixing the previous problem with the & symbol.
Several entries that should remain in English have been fixed.

### CODE REVIEW

File: resources/zh/cemu.po [hunk 1]
```diff
@@ -2770,87 +2774,87 @@ msgstr "地区"
...
 
 #: wxGameList.cpp
 msgid "Show &game time"
-msgstr "显示 &游戏时长"
+msgstr "显示 游戏时长 (&G)"
 
 #: wxGameList.cpp
 msgid "Show &last played"
-msgstr "显示 &上次游玩时间"
+msgstr "显示 上次游玩时间"
```
Comment (Fs00) [inline resources/zh/cemu.po:2853]: You forgot to add the mnemonics here.

**Review by Fs00 — COMMENTED** [review]
Thanks! There's only a small thing to address.

**Comment by Fs00** [comment]
Thanks for the quick fix!

**Comment by Genglxy** [comment]
> 感谢您的快速修复！

That' s the way it should be! You have checked very carefully. Great work and good cooperation!

**Pull request merged**